### ISSUE TITLE
Component should be use with material-ui

### ISSUE THREAD

**Issue by mustafaulukaya** [clinta74/react-dynamic-filterbar-bootstrap#18]
I think this component could work with material-ui 4 and material-ui 5.

**Comment by clinta74** [comment]
Yeah I had plans to create a version using mui soon. Which is why I split off the base as earlier. Did you have a specific need for version 4  or 5?
